package.json

```diff
@@ -38,18 +40,24 @@
     "@semantic-release/git": "^10.0.1",
     "@semantic-release/npm": "^10.0.4",
     "@semantic-release/release-notes-generator": "^11.0.4",
+    "@types/node": "^22.9.0",
```

Comment:
Can we use the same major version of node's types as the node version for the repo (`.nvmrc`)?